webrtc.html

```diff
@@ -8748,13 +8760,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </div>
       <div>
         <pre class="idl">dictionary RTCDataChannelInit {
-             boolean        ordered = true;
-             unsigned short maxPacketLifeTime;
-             unsigned short maxRetransmits;
-             USVString      protocol = "";
-             boolean        negotiated = false;
+             boolean         ordered = true;
+             unsigned short? maxPacketLifeTime = null;
+             unsigned short? maxRetransmits = null;
+             USVString       protocol = "";
+             boolean         negotiated = false;
              [EnforceRange]
-             unsigned short id;
+             unsigned short? id = null;
```

Comment:
https://heycam.github.io/webidl/#EnforceRange doesn't mention nullable types, but I think it'll work. Can you update https://github.com/w3c/web-platform-tests/blob/master/webrtc/RTCPeerConnection-createDataChannel.html in a companion PR to adjust the tests for the 3 nullable types? Test null and undefined for each.